Training/python/2018v1/Training_v0p1.py

```diff
@@ -31,10 +31,13 @@
 sys.path.insert(0, "..")
 from common import *
 import DataLoader
+import itertools
+import psutil
+
 
 class DeepTauModel(keras.Model):
 
-    def __init__(self, *args, loss=None, **kwargs):
+    def __init__(self, *args, loss=None, n_tau=None, use_AdvDataset=False, adv_parameter=[1,1], n_adv_tau=None, adv_learning_rate=None, **kwargs):
```

Comment:
`n_tau = tf.shape(x)[0] - n_adv_tau` => no need to pass `n_tau` in the constructor.